sim/paladin/soc.go

```diff
@@ -90,7 +90,7 @@ func (paladin *Paladin) registerSealOfCommand() {
 			SpellSchool: core.SpellSchoolHoly,
 			DefenseType: core.DefenseTypeMelee,
 			ProcMask:    core.ProcMaskMeleeMHSpecial | core.ProcMaskProc,
-			Flags:       core.SpellFlagMeleeMetrics,
+			Flags:       core.SpellFlagMeleeMetrics | SpellFlag_RV, // RV Worked on PTR
```

Comment:
What exactly does RV do? Looking at the wowhead entry it doesn't seem like seals should proc it?
<img width="408" alt="image" src="https://github.com/user-attachments/assets/883a19df-7e65-4900-bf5e-f03b8f5a7ed1">

Reply:
Seal of Command was added to the abilities that procced RV on the last week of the PTR as an undocumented change. Several Light Club members tested it to verify that is does proc it including myself.  This change was done after RV was functioning like ignite and rolling damage (pooling) so it seem unlikely it was an oversight. If it doesn't make it live we can always remove this flag. I suspect it was a late balancing change to bring Twisting up to par with fast 2H ret.